Source/SwiftyDropbox/Shared/Handwritten/OAuth/OAuth.swift

```diff
@@ -109,6 +115,12 @@ open class DropboxOAuthManager {
             return
         }
 
+        if usePKCE {
+            authSession = AuthSession(scopeRequest: scopeRequest)
```

Comment:
its unclear with the name `AuthSesssion` that that struct is specific to PKCE. it is also unclear when you use the word "code flow" (for eg) that you are referring to PKCE. Since it is a standard, and many users are just glancing at this codebase..I'd be crystal clear when code or flow refers to PKCE.

I 'd name that struct `OAuthPCKESession` or the like

Reply:
yeah, I can add PKC for clarity